blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
Since the major change in this endpoint is that each entry in the bundlelist has it's own timestamp and tenant, I would add a simple test for each of those.  Other than that, this PR looks ready to go.